tests/pytests/integration/states/test_npm.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+
+@pytest.fixture(scope="package", autouse=True)
+def install_npm(salt_cli, salt_minion):
+    result = salt_cli.run("pkg.install", "npm", minion_tgt=salt_minion.id)
```

Comment:
Since the install isn't part of the test, no need to go through the shelling out.
`sminion` can be used, it like a local client, faster. `create_sminion(). functions.pkg.install("npm")`
Grep our code for examples.

Reply:
Also, these could be functional tests, right? Also faster since it skips the communication protocol... Which, for this test, is not really required right?

Reply:
Converting them to functional tests is probably reasonable :+1:

Reply:
Done